kickstarts/base.ks.erb

```diff
@@ -167,169 +50,21 @@ set -x
 # Unset it so ruby_parser doesn't print LOTS of output.
 unset DEBUG
 
-###
-## Create the manageiq firewall zone
-###
-firewall-offline-cmd --new-zone=manageiq
-
-###
-# Set the manageiq firewall zone as default
-###
-firewall-offline-cmd --set-default-zone=manageiq
-
-###
-# Allow HTTP, HTTPS, SSH and postgresql in the manageiq firewall zone
-###
-firewall-offline-cmd --zone=manageiq --add-service=http
-firewall-offline-cmd --zone=manageiq --add-service=https
-firewall-offline-cmd --zone=manageiq --add-service=ssh
-firewall-offline-cmd --zone=manageiq --add-service=postgresql
-
-www_root="/var/www"
-app_root="$www_root/miq"
-appliance_root="/opt/manageiq/manageiq-appliance"
-
-rm -rf $app_root
-
-git clone <%= @appliance_checkout.remote %> $appliance_root
-pushd $appliance_root
-  git checkout <%= @appliance_checkout.branch %>
-  git reset --hard <%= @appliance_checkout.commit_sha %>
-popd
-
-mkdir -p $app_root
-git clone <%= @manageiq_checkout.remote %> $app_root/vmdb
-pushd $app_root/vmdb
-  git checkout <%= @manageiq_checkout.branch %>
-  git reset --hard <%= @manageiq_checkout.commit_sha %>
-popd
-
-# Symlink extracted repo (manageiq-appliance) to old /var/www/miq/system location.
-ln -vs $appliance_root $app_root/system
+<%= render_partial "post/firewalld" %>
 
-# Symlink old lib directory to gems/pending.
-ln -vs $app_root/vmdb/gems/pending $app_root/lib
+<%= render_partial "post/source_setup" %>
```

Comment:
We definitely need numbers in the names of these things as it will be hard to tell what files in the post section go in what order.  What do you think?